Automattic/wp-calypso
# Theme Discovery: Gate Theme Showcase Theme Card number behind Discovery feature flag

We need to gate the number of Theme Cards shown on the Theme Showcase behind the Discovery feature flag so that the Production Theme Showcase continues to show more Theme Cards per row until the Theme Discovery project gets validated.
## Proposed Changes

* We now show the same number of Theme Cards as in the Collection Carousel on the Theme Showcase only when the Discovery feature flag is enabled.

## Testing Instructions



* Open the Calypso Live Link
* Go to the LoTS and apply `?flags=themes/discovery-lots`
* You should see that the Theme Showcase displays the same number of Theme Cards as the Collection Carousel.
* Remove the flag by reloading or `?flags=-themes/discovery-lots`
* The number of Theme Cards shown on the Theme Showcase should be different (same as before these changes which is 4 or so by default and it increases as you increase the resolution).

Before without Feature Flag:
![Captura de pantalla 2023-10-31 a las 14 21 33](https://github.com/Automattic/wp-calypso/assets/1989914/bd0cdaeb-3f8c-45b4-a2d6-809a461bb2c2)

After without Feature Flag:
![Captura de pantalla 2023-10-31 a las 14 22 06](https://github.com/Automattic/wp-calypso/assets/1989914/7cb48453-5f54-4efc-a779-26afce126f9d)

## Pre-merge Checklist
